_2017_pages/cfp.html

```diff
@@ -15,102 +15,273 @@ <h2 class="first">Key Dates</h2>
       <p>Deadline for CFP: September 4th, 2020. Anywhere on Earth.</p>
       <p>Date of speaker content submission: TBD. Likely first week of October 2020.</p>
 
-      <h2 class="first">Format</h2>
+      <h2 class="first">Conference Summary</h2>
       <p>
       PyMCon 2020 is an asynchronous-first conference.
-      All accepted speakers will submit a video or code ahead of time,
-      which will be posted in a TBD place (probably YouTube).
+      All accepted speakers will submit a video with their presentation ahead of time.
       </p>
 
-      <h2 class="first">What we are looking for</h2>
       <p>
-        We are looking for folks like you to share your cool tricks, best practices,
-        and general knowledge with the Bayesian community.
-        This conference will have two tracks,
-        a "Conjugate Model" track for folks just getting started out,
-        and a "No U turn Sampler" track for folks that are already tuned and well warmed up with Bayesian Methods.
-        (Track names  and puns subject to change).
+      We are looking for folks like you to share your cool tricks, best practices,
+      general knowledge, or model build methods with the Bayesian community.
+      Presentations should have some relevance to the PyMC community, more details on
+      what we look for in a presentation and how to propose one can be found below.
       </p>
 
       <p>
-          There are three formats we're looking for specifically:
+      All presentations will be pre-recorded and will be shared on the PyMC discourse in
+      late October for the async component. We are also planning a synchronous component
+      for early November. Further details are below as well.
       </p>
 
-      <h3 class="first">Talks</h3>
-      <p class=normal-text>
-        Traditional talks are 30 to 40 minutes in length and should be pre-recorded.
-        If accepted, speakers are asked to submit a video file that contains the recording of their talk,
-        which will be uploaded and viewed prior to the conference.
-        As pre-recorded videos, talks are expected to be polished and crisp.
-        Slide contents and language should be professional and comprehendible.
-        A successful talk does not have to be about a novel, cutting-edge technology topic but should be distinctive and have the ability to keep the attention of the audience.
-        We will provide a platform for audience discussions and Q&A.
-        Accepted speakers are asked to be online the day of for at least one session of Q&A to answer questions live, which is 20 minutes long;
-        of course, they are free to participate in the discussions with the audience outside of this Q&A session as well.
+      <p>
+      This conference will have two tracks:
+      <ul>
+        <li>a "Conjugate Model" track for folks just getting started out,</li>
+        <li>and a "No U Turn Sampler" track for folks that are already  warmed up with
```

Comment:
should we use these names too in the google form for proposals?

Reply:
I mostly made these as jokes and was assuming someone was going to delete them cause theyre not very good jokes. But if we want to double down we can haha

Reply:
We probably shouldnt because you have to be really bayesian to get what it means and beginner and advanced are more straightforward haha